channeldb/db.go

```diff
@@ -270,6 +270,11 @@ func Open(dbPath string, modifiers ...OptionModifier) (*DB, error) {
 func resetChanStateCache(cache *kvdb.Cache) error {
 	cache.Wipe()
 
+	cache.SkipBucket(closedChannelBucket)
+	cache.SkipBucket(closeSummaryBucket)
+	cache.SkipBucket(fwdPackagesKey)
+	cache.SkipBucket(revocationLogBucket)
```

Comment:
With these skips added, are there still other non-fixed size buckets that are cached in-memory (and potentially causing OOM)?